### DISCUSSION TITLE
crazy documentation ideas...

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#100] [Ideas]
1 - Megazord single page index (maybe with "accordions", i.e. collapsing sections).

I have noticed that there is no `https://py5.ixora.io/reference/[index.html]` in the documentation website site namespace...
Could we have there a very short sentence on what is "the reference" and a huge summary with all the the categories and all the vocabulary from all sub-sections linked?

2 - Not so crazy... at the https://github.com/py5coding organization page, could we have a link here to the discussion forum and to the documentation pages?

**Comment by hx2A** [comment]
Great thoughts!

1. While setting up Jupyter book I noticed the absence of a URL https://py5.ixora.io/reference/ and tried to figure out how to do that. I couldn't get it to work but I forget why. I will take another look and see if I can do better.

2. I agree, those two links should be added. Are you able to edit them?? Or do you want to make a PR for that? Perhaps the permissioning is not set correctly. One of my todos is to setup the py5coding repos & organization a bit better.

**Reply by villares** [reply]
I think I might be able to edit here: https://github.com/py5coding/.github/tree/main/profile but I didn't want to overstep...

**Reply by villares** [reply]
https://github.com/py5coding/.github/pull/1

**Reply by hx2A** [reply]
Just approved it. Thank you!

**Comment by villares** [comment]
I think I'm struggling with consulting the positional argument order here at the `py5_tools.animated_gif()`, scrolling horizontally.

![image](https://user-images.githubusercontent.com/3694604/182032919-4ec86fd9-d55a-48d8-aba4-5fda9ca661f5.png)

Could we have each parameter name in its own line for those long signatures?

**Reply by hx2A** [reply]
You are referring to the Syntax section? I agree, this has gotten on my nerves also. I'll create an issue for this.
